oauth2server/proj/settings/default.py

```diff
@@ -81,4 +81,108 @@
         'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
     )
-}
\ No newline at end of file
+}
+
+
+# Custom logging to give us unique logging to work on remote containers and normal logging
+# Requirements for our custom loggers are:
+# 1)    A console logger that logs debug to console for DEBUG=TRUE and WARNINGS and higher when DEBUG=FALSE.
+# 2)    Disable all email sent to ADMINS via the mail_admins handler for the django.request and django.security loggers.
+# 3)    Add two custom logging formatters to enhance the logging output.
+# 4)    Add handlers to write files for 'proj' and 'app' name space. For production (DEBUG=FALSE) and dev (DEBUG=TRUE).
+# 5)    Add LOGGERS for 'apps' and 'proj' name space. Add LOGGERS for python and django default. Add a logger for remote
+#       files.
+
+LOGGING = {
```

Comment:
Can the format align with the SDC cross service patterns for leveraging the Splunk aggregator